.travis.yml

```diff
@@ -93,7 +93,7 @@ before_install:
       end=$((id * number_per_job));
       start=$((end - number_per_job));
 
-      liststr=tests/integration/targets/azure_rm_*/tasks/main.yml;
+      liststr=tests/integration/targets/azure_rm_rediscache/tasks/main.yml;
```

Comment:
azure_rm_*

Reply:
for testing purpose, will  change it back after tests done

Reply:
i think we it's a good moment to upgrade tests, so only updated, changed modules are tested.